scripts/train/train.py

```diff
@@ -169,10 +169,7 @@ def build_dataloader(cfg: DictConfig, tokenizer: PreTrainedTokenizerBase,
         raise ValueError(f'Not sure how to build dataloader with config: {cfg}')
 
 
-def main(cfg: DictConfig):
-    # Check for incompatibilities between the model and data loaders
-    validate_config(cfg)
```

Comment:
BLOCKING COMMENT:

`validate_config(cfg)` should still be called.

Reply:
➕ thanks for catching

Reply:
thanks; fixed this